requirements-extras.txt

```diff
@@ -7,7 +7,7 @@ pygal>=2.0.0
 typogrify>=2.0.4
 phpserialize>=1.3
 webassets>=0.10.1
-ipython[notebook]>=2.0.0
+jupyter>=1.0.0
```

Comment:
`notebook` should be the dependency... jupyter "metapackage" contain a lot of stuff which is not a dependency for the functionality...